README.md

```diff
@@ -1,6 +1,6 @@
 # LeaderboardProject
 
-This project is about setting up leaderboard
+This project is Leaderoard final touch by adding styles
 
 # Built With: 
 - HTML
```

Comment:
- [ ] Great job so far with your readme file, but I think there still room for improvement, your readme file is still missing some important sections like; `license and acknowledgement sections`. Please it will also be nice to update the `Getting started` section with explicit steps on how to clone and run your app locally as this will be of help to non technical users and some are potential recruiters. Checkout this [template](https://github.com/microverseinc/readme-template) from Microverse and adapt it to your project. Your readme file gives the first impression about the type of programmer your are, so please give it your best.